extras/help_generator/examples.sh

```diff
@@ -1,155 +0,0 @@
-#!/bin/bash
```

Comment:
Why are the files in help_generator removed?

Reply:
@steffengraber commented in #1313 that these three files could be removed. It helps a bit because this means there is less Py2 code to convert to Py3. The bash script is incidental, to not leave dangling threads.